input/pagecontent/iti-119.md

```diff
@@ -19,24 +19,28 @@ a patient participating in the Swiss EPR.
 
 ### Messages
 
-<div>{% include PDQm_ActorDiagram_ITI-78.svg %}</div>
+<div>{% include PDQm_ActorDiagram_ITI-119.svg %}</div>
 
-#### Query Patient Resource message
+#### Match Patient Resource Message
 
 ##### Message Semantics
 
-The semantics of the _Query Patient Resource_ message is the same as defined in
-[2:3.78.4.1.2](https://profiles.ihe.net/ITI/PDQm/ITI-78.html#2378412-message-semantics), with the following restrictions:
+The semantics of the _Match Patient Resource_ message is the same as defined in
+[2:3.119.4.1.2](https://profiles.ihe.net/ITI/PDQm/ITI-119.html#23119412-message-semantics), with the following restrictions:
 
-- The `telecom` search parameter SHALL NOT be used.
+- When specifying the count parameter, no more than 5 matches SHALL be requested.
+
+The HTTP body SHALL conform to the [PDQm Match Input Parameters profile](StructureDefinition-CHPDQmMatchParametersIn.html).
 
 ##### Expected Actions
 
 The _Patient Demographics Supplier_ shall follow the expected actions as defined in
-[2:3.78.4.1.3](https://profiles.ihe.net/ITI/PDQm/ITI-78.html#2378413-expected-actions), with the following restrictions:
+[2:3.119.4.1.3](https://profiles.ihe.net/ITI/PDQm/ITI-119.html#23119413-expected-actions), with the following restrictions:
 
-- If there are more than 5 matches, the result should return zero matches with an OperationOutcome requesting more
+- If there are more than 5 matches, the result should return zero matches with an OperationOutcome requesting more 
   query parameters.
+- If the client requests more than 5 matches, the result should return zero matches with an OperationOutcome 
+  explaining the server can't fulfill the request.
 
```

Comment:
- Only Patient identifiers from the community (MPI-PID and EPR-SPID) SHALL be returned.

Reply:
Added.